apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/responses/components/ResponseDataView.tsx

```diff
@@ -55,7 +56,10 @@ export const formatContactInfoData = (responseValue: TResponseDataValue): Record
 export const extractResponseData = (response: TResponseWithQuotas, survey: TSurvey): Record<string, any> => {
   const responseData: Record<string, any> = {};
 
-  for (const question of survey.questions) {
+  // Derive questions from blocks
+  const questions = getQuestionsFromBlocks(survey.blocks);
```

Comment:
When we renamed `question` to `element` I haven't thought about how often we have `question` in the codebase and how this leads to inconsistencies. I think it's fine for now, but can be confusing in the future that we still talk about `questions` instead of `elements` in so many places.

Reply:
yeah I also thought about this, ideally this should be fixed when we finally refactor the code as well but for the PoC I think it should be fine as long as everything works properly